spec/functional/win32/security_spec.rb

```diff
@@ -31,14 +34,36 @@
   # support for windows.
   #
   # TODO - Add negative tests once mixlib-shellout has user support
-  it "has_admin_privileges? returns false when running as non-admin" do
-    skip "requires user support in mixlib-shellout"
+  describe "running as non admin user" do
+    let(:user) { "security_user" }
+    let(:password) { "security" }
+    before do
+      include Chef::Mixin::WideString
```

Comment:
you might just want to pull the mixin from #5832 in here, and then you can use the other tests there as a guide.

`with_user_context` yields a block, so you could pass a block with `Chef::ReservedNames::Win32::Security.has_admin_privileges?` and then check the return of the block for true/false.